packages/mongo-menu/src/MongoMenu.tsx

```diff
@@ -255,9 +261,10 @@ function MongoMenu({
         <FocusableMenuItem>
           <Button
             size="small"
-            href={accountURL}
+            href={accountURL || undefined}
             className={accountButtonStyle}
-            as="a"
+            as={accountURL ? 'a' : 'button'}
```

Comment:
The Button component *should* handle disabled states when rendered as a link. I think this change is unnecessary.

Reply:
We can change this back to an `a` tag if you insist, but this isn't updated because of my concern over the component, but rather clarity and accessibility. Justin brought this up way back when web chapter was just being born (or maybe before?), but I was taught by him and other sources:
- **only** use an `a` tag if you are going to set an `href` and not modify any behavior
- if you have no `href` tag, if you interrupt the call with a JS handler, if you directly manipulate `window.location`, etc, then you should **always** use a `button` tag instead, so that no user (especially on a screen reader) experiences an unexpected behavior.

This is a picky opinion I follow pretty religiously, but if that goes against the accessibility / code style / etc standards for this repo, I have no problem rolling things back to always be an `a`.

(aside: as a bonus if it is a button—we can very easily test the `disabled` prop in the unit tests)

Reply:
@brookescarlett Might be able to make a better call of what we should do here than I in this case.

Reply:
@seanmhanson's logic makes sense to me

Reply:
Sounds good, seems fine then.